### ISSUE TITLE
Templates ticket don't apply with the entity

### ISSUE THREAD

**Issue by SebBpod** [glpi-project/glpi#10837]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.0

### Bug description

Hello,

When I create a ticket with an entity that has a template affiliated to it, nothing happens. The ticket page is not filled in. I have not changed anything since yesterday and the bug appeared today.

### Relevant log output

```shell
I don't found any error on the log
```

### Steps To reproduce

Create new ticket
Choose a entity
The page automatically reloads with the template of the entity

### Anything else?

Notice: Entity config "tickettemplates_id" should be get using its reference field "tickettemplates_strategy" with a "0" default value in /home/clients/d9439d945594dd80a9427f3204c69be3/web/glpi/src/Entity.php on line 3174

I have this message with a tech profil but not with the admin profil

**Comment by cconard96** [comment]
Which GLPI 10 release are you using?
Please test with the latest RC release.
https://github.com/glpi-project/glpi/releases/tag/10.0.0-rc2

**Comment by SebBpod** [comment]
Yes i'm using the GLPI 10 release, i will try this and I give you a feedback

**Comment by SebBpod** [comment]
> Yes i'm using the GLPI 10 release, i will try this and I give you a feedback

sorry bad answer, i'm using the GLPI 10.0.0-dev  version

**Comment by cconard96** [comment]
I'm not able to recreate the issue. It is possible that the profile has a template defined.
The entity's defined template is only used when no template is specified in the user's current profile. Also note that if you change the template in either place, you may need to log out and back in to see the changes.

**Comment by cconard96** [comment]
This may be fixed by #11421
